wsktools/WhiskSwiftTools/ProjectReader.swift

```diff
@@ -498,8 +498,9 @@ open class ProjectReader {
                     if let item = actionsDict[(prefix+itemName) as NSString] {
                         var runtime = item.runtime
                         if let kind = action["kind"] as? String {
-                            if runtime == Runtime.swift3 && kind == "swift:3" {
-                                runtime = Runtime.swift3
+                            // swift:default == swift:3 at the moment
+                            if runtime == Runtime.swift && kind == "swift:3" {
+                                runtime = Runtime.swift
```

Comment:
I'm not sure I understand the purpose of this. If `runtime` is already `Runtime.swift` then why assign it (again) to `Runtime.swift`? This seems either unnecessary or is a bug that is meant to accomplish something else. Or... I'm just reading it wrong 😝